mappings/net/minecraft/world/gen/feature/StructureFeature.mapping

```diff
@@ -0,0 +1,117 @@
+	FIELD f_dtdybwen biomes Lnet/minecraft/unmapped/C_odfnijdo;
+	FIELD f_gzmkaukp generationStep Lnet/minecraft/unmapped/C_rdrmebyw$C_eibovkzt;
+	FIELD f_naccwevr DIRECT_CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_rfftwmba CODEC Lcom/mojang/serialization/Codec;
+	FIELD f_syxyfijt adaptNoise Z
+	FIELD f_uigybjjg spawnOverrides Ljava/util/Map;
+	METHOD <init> (Lnet/minecraft/unmapped/C_odfnijdo;Ljava/util/Map;Lnet/minecraft/unmapped/C_rdrmebyw$C_eibovkzt;Z)V
+		ARG 1 biomes
+		ARG 2 spawnOverrides
+		ARG 3 generationStep
+		ARG 4 adaptNoise
+	METHOD m_dckijrrf getType ()Lnet/minecraft/unmapped/C_yaozqavy;
+	METHOD m_dozyucdf adjustBoundingBox (Lnet/minecraft/unmapped/C_hmqypqzz;)Lnet/minecraft/unmapped/C_hmqypqzz;
+		ARG 1 box
+	METHOD m_egvfbazt generate (Lnet/minecraft/unmapped/C_wqxmvzdq;Lnet/minecraft/unmapped/C_kgfalbim;Lnet/minecraft/unmapped/C_ajguowya;Lnet/minecraft/unmapped/C_edfiqifk;Lnet/minecraft/unmapped/C_zxhjfyir;JLnet/minecraft/unmapped/C_ynrszrtu;ILnet/minecraft/unmapped/C_qpninoyb;Ljava/util/function/Predicate;)Lnet/minecraft/unmapped/C_efiavfys;
+		ARG 1 registryManager
+		ARG 2 chunkGenerator
+		ARG 3 biomeSource
+		ARG 4 randomState
+		ARG 5 structureTemplateManager
+		ARG 6 seed
+		ARG 8 chunkPos
+		ARG 10 world
+		ARG 11 validBiomePredicate
+	METHOD m_etbsnvzz getSpawnOverrides ()Ljava/util/Map;
+	METHOD m_jlyvhzqo isValidBiome (Lnet/minecraft/unmapped/C_quklzgtb$C_tbkxgvzz;Lnet/minecraft/unmapped/C_kgfalbim;Lnet/minecraft/unmapped/C_edfiqifk;Ljava/util/function/Predicate;)Z
+		ARG 0 generationStub
+		ARG 1 chunkGenerator
+		ARG 2 randomState
+		ARG 3 validBiomePredicate
+	METHOD m_nhpcolbr codec (Lcom/mojang/serialization/codecs/RecordCodecBuilder$Instance;)Lcom/mojang/datafixers/Products$P4;
+		ARG 0 instance
+	METHOD m_nrfeerxd shouldAdaptNoise ()Z
+	METHOD m_nyuiohlj getLowestHeight (Lnet/minecraft/unmapped/C_quklzgtb$C_ooyswlix;II)I
+		ARG 0 context
+		ARG 1 x
+		ARG 2 z
+	METHOD m_qlvjonmq getGenerationStep ()Lnet/minecraft/unmapped/C_rdrmebyw$C_eibovkzt;
+	METHOD m_tfytmvqt getBiomes ()Lnet/minecraft/unmapped/C_odfnijdo;
+	METHOD m_tgkjthyd findGenerationPos (Lnet/minecraft/unmapped/C_quklzgtb$C_ooyswlix;)Ljava/util/Optional;
+		ARG 1 context
+	METHOD m_tksnmdcg getCornerHeights (Lnet/minecraft/unmapped/C_quklzgtb$C_ooyswlix;IIII)[I
+		ARG 0 context
+		ARG 1 x
+		ARG 2 width
+		ARG 3 z
+		ARG 4 height
+	METHOD m_xicdrobb postPlace (Lnet/minecraft/unmapped/C_ldkphtbr;Lnet/minecraft/unmapped/C_rhdulivw;Lnet/minecraft/unmapped/C_kgfalbim;Ljava/util/Random;Lnet/minecraft/unmapped/C_hmqypqzz;Lnet/minecraft/unmapped/C_ynrszrtu;Lnet/minecraft/unmapped/C_ccebncrd;)V
+		ARG 1 world
+		ARG 2 structureManager
+		ARG 3 chunkGenerator
+		ARG 4 random
+		ARG 5 boundingBox
+		ARG 6 chunkPos
+		ARG 7 children
+	METHOD m_zswgnsnz onTopOfChunkCenter (Lnet/minecraft/unmapped/C_quklzgtb$C_ooyswlix;Lnet/minecraft/unmapped/C_ybztoekc$C_wkdeqzbn;Ljava/util/function/Consumer;)Ljava/util/Optional;
```

Comment:
```suggestion
	METHOD m_zswgnsnz createStubOnChunkCenter (Lnet/minecraft/unmapped/C_quklzgtb$C_ooyswlix;Lnet/minecraft/unmapped/C_ybztoekc$C_wkdeqzbn;Ljava/util/function/Consumer;)Ljava/util/Optional;
```
This is a factory (pointlessly wrapped in `Optional`?) for a `StructureFeature.GenerationStub`, and `onTopOfChunkCenter` doesn't seem to indicate that.